SAP AS JAVA documentation (#120205)
Azure/azure-content: articles/sentinel/sap/sap-solution-security-content.md

| **SAP - Multiple Logons from the same IP** | Identifies the sign-in of several users from same IP address within a scheduled time interval.   <br><br>**Sub-use case**: [Persistency](#persistency) | Sign in using several users through the same IP address. <br><br>**Data sources**: SAPcon - Audit Log | Initial Access |
| **SAP - Multiple Logons by User** | Identifies sign-ins of the same user from several terminals within scheduled time interval.  <br><br>Available only via the Audit SAL method, for SAP versions 7.5 and higher. | Sign in using the same user, using different IP addresses.   <br><br>**Data sources**: SAPcon - Audit Log | PreAttack, Credential Access, Initial Access, Collection <br><br>**Sub-use case**: [Persistency](#persistency) |
| **SAP - Informational - Lifecycle - SAP Notes were implemented in system** | Identifies SAP Note implementation in the system. | Implement an SAP Note using SNOTE/TCI. <br><br>**Data sources**: SAPcon -  Change Requests | - |

> | **(Preview) SAP - AS JAVA - Sensitive Privileged User Logged in** | Identifies instances of dialog logon (UME) by sensitive privileged users within the SAP AS Java environment. It is crucial to maintain the list of privileged users in the "SAP - Privileged Users" Watchlist.| Logon to the backend system using privileged users. <br><br>**Data sources**: SAPcon - SAPJAVAFilesLog | Valid Accounts |

Reviewer: ```suggestion
| **(Preview) SAP - AS JAVA - Sensitive Privileged User Signed In** | Identifies instances of dialog sign-in (UME) by sensitive privileged users within the SAP AS Java environment. You must maintain the list of privileged users in the "SAP - Privileged Users" Watchlist.| Sign in to the backend system using privileged users. <br><br>**Data sources**: SAPcon - SAPJAVAFilesLog | Valid Accounts |
```

> | **(Preview) SAP - AS JAVA - Login from unexpected network** | Identifies logons from an unexpected network. | Logon to the backend system from an IP address which is not assigned to one of the networks in the `SAP - Networks` watchlist. <br><br>**Data sources**: SAPcon - SAPJAVAFilesLog | Lateral Movement- Valid Accounts |

Reviewer: ```suggestion
| **(Preview) SAP - AS JAVA - Sign In from Unexpected Network** | Identifies sign-ins from an unexpected network. | Sign in to the backend system from an IP address that isn't assigned to one of the networks in the `SAP - Networks` watchlist. <br><br>**Data sources**: SAPcon - SAPJAVAFilesLog | Lateral Movement- Valid Accounts |
```

| **SAP - Execution of Obsolete or Insecure Function Module** |Identifies the execution of an obsolete or insecure ABAP function module. <br><br>Maintain obsolete functions in the [SAP - Obsolete Function Modules](#modules) watchlist. Make sure to activate table logging changes for the `EUFUNC` table in the backend. (SE13)<br><br> **Note**: Relevant for production systems only. | Run an obsolete or insecure function module directly using SE37. <br><br>**Data sources**: SAPcon -  Table Data Log | Discovery, Command and Control |
| **SAP - Execution of Obsolete/Insecure Program** |Identifies the execution of an obsolete or insecure ABAP program. <br><br> Maintain obsolete programs in the [SAP - Obsolete Programs](#programs) watchlist.<br><br> **Note**: Relevant for production systems only. | Run a program directly using SE38/SA38/SE80, or by using a background job.  <br><br>**Data sources**: SAPcon -  Audit Log | Discovery, Command and Control |
| **SAP - Multiple Password Changes by User** | Identifies multiple password changes by user. | Change user password <br><br>**Data sources**: SAPcon - Audit Log | Credential Access |

> | **(Preview) SAP - AS JAVA - User creates and uses new user** |  Identifies the creation or manipulation of users by admins within the SAP AS Java environment. | Logon to the backend system using users that you have created or manipulated. <br><br>**Data sources**: SAPcon - SAPJAVAFilesLog | Create Account, Valid Accounts |

Reviewer: ```suggestion
| **(Preview) SAP - AS JAVA - User creates and uses new user** |  Identifies the creation or manipulation of users by admins within the SAP AS Java environment. | Sign in to the backend system using users that you created or manipulated. <br><br>**Data sources**: SAPcon - SAPJAVAFilesLog | Create Account, Valid Accounts |
```

> | **(Preview) SAP - AS JAVA - User creates and uses new user** |  Identifies the creation or manipulation of users by admins within the SAP AS Java environment. | Logon to the backend system using users that you have created or manipulated. <br><br>**Data sources**: SAPcon - SAPJAVAFilesLog | Create Account, Valid Accounts |

Reviewer: ```suggestion
| **(Preview) SAP - AS JAVA - User Creates and Uses New User** |  Identifies the creation or manipulation of users by admins within the SAP AS Java environment. | Sign in to the backend system using users that you have created or manipulated. <br><br>**Data sources**: SAPcon - SAPJAVAFilesLog | Create Account, Valid Accounts |
```
